esx_service/vmci/vmci_client.c

```diff
@@ -267,6 +269,9 @@ vsock_get_reply(be_sock_id *s, be_request *r, be_answer* a)
    b = 0;
    ret = recv(s->sock_id, &b, sizeof b, 0);
    if (ret == -1 || ret != sizeof b ) {
+      if (a->errBuf) {
+         snprintf(a->errBuf, 500, "Failed to receive magic data: received %d expected %d\n", ret, sizeof b);
```

Comment:
Please do not use literals for sizes.  A #define (in C) and const (in Go) please